src/back/controllers/api/routes/challenge/getOne.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const challenge = require('../../models/challenge');
+
+const getOne = (req, res) => {
+    challenge.findOne({ _id: req.params.id }, (err, challengeDetails) => {
```

Comment:
Agree with @borys-ihnatyev on errors handlers, but I'm not sure we need an additional layer here, as for me it is too pre-emptive for this situation.